### ISSUE TITLE
emldown functionality not available in > 1.6.0 versions ?

### ISSUE THREAD

**Issue by yvanlebras** [earnaud/MetaShARK-v2#179]
It seems there is no more folder with emldown results on the downloadable data packages, once EML generated by MetaShARK.

**Comment by earnaud** [comment]
Yup, {emldown} is not maintained anymore, and didn't pass some version of R (4.1.0 if I remember well).
